### DISCUSSION TITLE
wave 2.13 model G - flickering

### DISCUSSION THREAD

**Discussion by VoIP-Engineer** [jayofelony/pwnagotchi#127] [Q&A] [answered]
Hey,

I managed to get my e-ink screen to work but it is blinking and flashing pretty much all the time, like its redrawing 500 times a second, then it will be ok for a few min then will go nuts again,

Anyone have an ideas?

**Comment by jayofelony** [comment]
Get a different display, its a 4 color display that refreshes every color each cycle.

**Reply by VoIP-Engineer** [reply]
So no chance of a quick fix :(

**Reply by jayofelony** [reply, marked as answer]
No fix at all, this is how this display works sadly.

**Reply by VoIP-Engineer** [reply]
Thanks for the help
